src/v/storage/kvstore.h

```diff
@@ -77,16 +77,19 @@ struct kvstore_config {
     std::chrono::milliseconds commit_interval;
     ss::sstring base_dir;
     debug_sanitize_files sanitize_fileops;
+    ss::sstring metrics_name;
 
     kvstore_config(
       size_t max_segment_size,
       std::chrono::milliseconds commit_interval,
       ss::sstring base_dir,
-      debug_sanitize_files sanitize_fileops)
+      debug_sanitize_files sanitize_fileops,
+      ss::sstring metrics_name = "")
```

Comment:
hmm.... ? i don't follow this. you _either_ set it or not, why have this empty name ? 

this change doesn't make sense to me. why not define it in the 2 places it's used.